### ISSUE TITLE
BUG BASH: Student (kiki and MOHIT)- student profile

### ISSUE THREAD

**Issue by cpg00** [adilsaju/yoke#53]
Lack of information (alt text showing instead of picture, no course or student id) (FOR KIKI), images are divided in two lines, should be one line only.
No gifs, limit file format, or just don`t use them for example, since they are not acceptable file format for business.
UPLOAD DOCUMENTS:
- again not in the same line, wrong use of icon for scan buttom.
- There is no upload confimation.

**Comment by adilsaju** [comment]
It will be better if you add a screenshot please

**Comment by adilsaju** [comment]
Please add steps
To reproduce

**Comment by cpg00** [comment]
STEPS: Loged in using student's credential, open student profile page

@adilsaju Bug was encountered through @misserism laptop, would need more days to get a screenshot

**Comment by adilsaju** [comment]
Gifs won't be used!

**Comment by adilsaju** [comment]
Toast message added
